src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,108 @@
+@import "./utils/_variables";
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: $card-width;
+  padding: 16px;
+  padding-top: 32px;
+  border: 1px solid $base-border-color;
+  border-radius: 5px;
+  transition: all $transition-time;
```

Comment:
Don't use `transition: all`, because this way it's easier to animate something by mistake. Write all of the properties you want to transition.